### PR TITLE
Clarify the context of the integration setup guides

### PR DESCRIPTION
**Pull request by pscheit**
image

### CODE REVIEW

**Comment by pscheit** [comment]
If we would want to add that clarifying paragraph on top for gitlab, I would add one for github + bitbucket as well.

added the small other wished from @naderman , too

**Comment by glaubinix** [comment]
Looks good. Feel free to add the paragraph to the other integrations too.

**Comment by pscheit** [comment]
we used callout warning here
image

shall we
a) use callout warning everywhere
b) leave it there and have just a paragraph everywhere else
c) convert this warning to a paragraph to have it consistent?

**Comment by glaubinix** [comment]
I would leave this as is. Bitbucket is a really different to GitHub and GitLab. They have different solutions for self-hosted vs cloud and the callout up there is the most important thing on that docs page. In 99% of all cases, you are trying to connect to a self-hosted instance of Bitbucket and not bitbucket.org.

**Review by glaubinix — APPROVED** [review]

File: docs/self-hosted/bitbucket-integration-setup.md [hunk 1]
```diff
@@ -8,7 +8,11 @@
 </div>
 
 ## Initial Setup
-Hit the “Add integration“ button on the admin page to get to the form below. To setup a Bitbucket Integration with Private Packagist start by selecting "Bitbucket" as the platform and enter <i>https://bitbucket.org</i> as the base URL, as seen in the example below.
+Hit the “Add integration“ button on the admin page to get to the form below.
+
+![Add Integration](/Resources/public/img/docs/self-hosted/08-integration.png)
+
+To setup a Bitbucket Integration with Private Packagist start by selecting "Bitbucket" as the platform and enter <i>https://bitbucket.org</i> as the base URL, as seen in the example below.
```
Comment (naderman) [inline docs/self-hosted/bitbucket-integration-setup.md:15]:
```suggestion
To setup a Bitbucket integration with Private Packagist start by selecting "Bitbucket" as the platform and enter <i>https://bitbucket.org</i> as the base URL, as seen in the example below.
```

was already wrong before, but worth fixing ;-)

File: docs/self-hosted/bitbucket-integration-setup.md [hunk 2]
```diff
@@ -8,7 +8,11 @@
 </div>
 
 ## Initial Setup
-Hit the “Add integration“ button on the admin page to get to the form below. To setup a Bitbucket Integration with Private Packagist start by selecting "Bitbucket" as the platform and enter <i>https://bitbucket.org</i> as the base URL, as seen in the example below.
+Hit the “Add integration“ button on the admin page to get to the form below.
```
Comment (naderman) [inline docs/self-hosted/bitbucket-integration-setup.md:11]:
```suggestion
Hit the “Add Integration“ button on the admin page to get to the form below.
```

File: docs/self-hosted/bitbucket-server-integration-setup.md [hunk 3]
```diff
@@ -1,8 +1,15 @@
 # Bitbucket Data Center / Server Integration Setup
 ## Private Packagist Self-Hosted
 
+This guide explains how to setup an OAuth integration for Private Packagist Self-Hosted with Bitbucket Data Center / Server integration.  
+If you are using our cloud product at packagist.com, [use this guide](../cloud/bitbucket-server-integration-setup.md).
+
 ## Initial Setup
-Hit the “Add integration“ button on the admin page to get to the form below. To setup a Bitbucket Data Center / Server integration with Private Packagist start by selecting "Bitbucket Data Center / Server" as the platform and enter the URL of your on-premise Bitbucket Server into the base URL field as seen in the example below.
+Hit the “Add integration“ button on the admin page to get to the form below.
```
Comment (naderman) [inline docs/self-hosted/bitbucket-server-integration-setup.md:8]:
```suggestion
Hit the “Add Integration“ button on the admin page to get to the form below. 
```

File: docs/self-hosted/github-integration-setup.md [hunk 4]
```diff
@@ -1,8 +1,15 @@
 # GitHub (Enterprise) Integration Setup
 ## Private Packagist Self-Hosted
 
+This guide explains how to setup an OAuth integration for Private Packagist Self-Hosted with either the on-premises version of GitHub, or their public service on github.com.  
+If you are using our cloud product at packagist.com, [use this guide](../cloud/github-integration-setup.md).
+
 ## Initial Setup
-Hit the “Add integration“ button on the admin page to get to the form below. To setup a GitHub integration with Private Packagist start by selecting "GitHub" as the platform and enter the URL of your GitHub Enterprise Server or <i>https://github.com</i> to use the public GitHub server as seen in the example below. A link to setup the OAuth application on GitHub will automatically be displayed.
+Hit the “Add integration“ button on the admin page to get to the form below.
```
Comment (naderman) [inline docs/self-hosted/github-integration-setup.md:8]:
```suggestion
Hit the “Add Integration“ button on the admin page to get to the form below.
```

File: docs/self-hosted/gitlab-integration-setup.md [hunk 5]
```diff
@@ -1,8 +1,15 @@
 # GitLab Integration Setup
 ## Private Packagist Self-Hosted
 
+This guide explains how to setup an OAuth integration for Private Packagist Self-Hosted with either the on-premises version of GitLab, or their public service on gitlab.com.  
+If you are using our cloud product at packagist.com, [use this guide](../cloud/gitlab-integration-setup.md).
+
 ## Initial Setup
-Hit the “Add integration“ button on the admin page to get to the form below. To setup a GitLab integration with Private Packagist start by selecting "GitLab" as the platform and enter the URL of your own GitLab server or use <i>https://gitlab.com</i> for the public GitLab server as seen in the example below. A link to setup the oauth application on GitLab will automatically be displayed.
+Hit the "Add integration" button on the admin page in Private Packagist Self-Hosted to get to the form below.
```
Comment (naderman) [inline docs/self-hosted/gitlab-integration-setup.md:8]:
```suggestion
Hit the "Add Integration" button on the admin page in Private Packagist Self-Hosted to get to the form below. 
```

**Review by naderman — CHANGES_REQUESTED** [review]

**Pull request merged**